src/AddressChecker.js

```diff
@@ -32,7 +35,7 @@ class AddressChecker extends Component {
       done: false,
       running: false,
       segwit: false,
-      coin: "0"
+      coin: "1"
```

Comment:
this was set as bitcoin but the hd path was bitcoin test, so set the the default to bitcoin test to be in sync